src/frontend/screens/Settings/components/Tools/index.tsx

```diff
@@ -70,14 +70,15 @@ export default function Tools() {
     // Prevent default behavior (Prevent file from being opened)
     ev.preventDefault()
 
-    if (ev.dataTransfer.items) {
-      // Use DataTransferItemList interface to access the file(s)
-      // If dropped items aren't files, reject them
-      if (ev.dataTransfer.items[0].kind === 'file') {
-        const exe = ev.dataTransfer.items[0].getAsFile()?.path
-        if (exe) {
-          return callTools('runExe', exe)
-        }
+    if (!ev.dataTransfer.items[0]) {
+      return
+    }
+    // Use DataTransferItemList interface to access the file(s)
+    // If dropped items aren't files, reject them
```

Comment:
just a reminder here to remove this Drag functionality. I don't think it is pretty useful. Maybe let's just add a new button like the others and select the EXE on click.